### PR TITLE
Fix slack channel on website

### PR DESCRIPTION
**Pull request by mlieberman85**

### CODE REVIEW

File: website/content/_index.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ technical domains.
 
 - [TAG Charter](https://github.com/cncf/toc/blob/main/tags/app-delivery.md)
```
Comment (TheFoxAtWork) [inline website/content/_index.md:22]:
```suggestion
- [TAG Charter](https://github.com/cncf/toc/blob/main/tags/security.md)
```
I also noticed the liaisons on that page are out of date :)

File: website/content/_index.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ technical domains.
...
 - [Community events](https://community.cncf.io/tag-app-delivery/)
-- Slack channel: [#tag-app-delivery](https://cloud-native.slack.com/messages/CL3SL0CP5)
+- Slack channel: [#tag-security](https://cloud-native.slack.com/archives/CDJ7MLT8S)
     - [Invite yourself to the CNCF Slack](https://slack.cncf.io/)
 - [Mailing list](https://lists.cncf.io/g/cncf-tag-app-delivery/topics)
```
Comment (TheFoxAtWork) [inline website/content/_index.md:26]:
```suggestion
- [Mailing list](https://lists.cncf.io/g/cncf-tag-security/topics)
```

**Review by TheFoxAtWork — COMMENTED** [review]
There are additional changes that will need made ( see attached) 

image

File: website/content/_index.md (hunk 2 above)
Comment (mlieberman85) [reply website/content/_index.md:26]: Thanks!

**Review by mnm678 — APPROVED** [review]

**Pull request merged**